files/default/slack_handler_webhook.rb

```diff
@@ -80,7 +80,12 @@ def slack_message(message, webhook, text_attachment = nil)
     http.verify_mode = OpenSSL::SSL::VERIFY_NONE
     req = Net::HTTP::Post.new(uri.path, 'Content-Type' => 'application/json')
     req.body = request_body(message, text_attachment)
-    http.request(req)
+    res = http.request(req)
+    # responses can be:
+    # "Bad token"
+    # "invalid_payload"
+    # "ok"
+    raise res.body unless res.body == 'ok'
```

Comment:
Curious about the impact of this. Does it mean the Chef deploy will now fail if the notification fails?